conans/test/integration/command/config_test.py

```diff
@@ -164,3 +166,16 @@ def test_init_overwrite(self):
     def _assert_dict_subset(self, expected, actual):
         actual = {k: v for k, v in actual.items() if k in expected}
         self.assertDictEqual(expected, actual)
+
+    @parameterized.expand([("", True), ("--raw", False)])
```

Comment:
Please recall to write new tests in pytest framework better.

Reply:
Sure, actually I'll do better, I can update this file to pytest on a next PR, so I don't mess this one.